### PR TITLE
Modified comments on upper.py ,made it more clean

### PR DESCRIPTION
**Pull request by sethusankar**
### **Describe your change:**

fixes: #3807 

This is just a word change in the comments section to make the comments look more readable. No other issues are expected as a part of this change.

* [ ] Add an algorithm?
* [ ] Fix a bug or typo in an existing algorithm?
* [X] Documentation change?

### **Checklist:**
* [X] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [X] This pull request is all my own work -- I have not plagiarized.
* [x] I know that pull requests will not be merged if they fail the automated tests.
* [X] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [X] All new Python files are placed inside an existing directory.
* [X] All filenames are in all lowercase characters with no spaces or dashes.
* [X] All functions and variable names follow Python naming conventions.
* [X] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [x] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.

* [x]  All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation

* [x] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{3807}`.

### CODE REVIEW

**Review by sethusankar — COMMENTED** [review]
This should fix issue 3807

**Comment by xcodz-dot** [comment]
add the following line to your PR body `fixes: #3807` to link it to that issue

File: strings/upper.py [hunk 1]
```diff
@@ -13,7 +13,7 @@ def upper(word: str) -> str:
     """
 
     # converting to ascii value int value and checking to see if char is a lower letter
-    # if it is a capital letter it is getting shift by 32 which makes it a capital case
+    # if it is a lower letter it is getting shift by 32 which makes it a capital case
```
Comment (xcodz-dot) [inline strings/upper.py:16]:
```suggestion
    # if it is a lowercase letter it is getting shift by 32 which makes it a capital case
```
grammatical fix ('lower is not a good word to explain')

**Review by sethusankar — COMMENTED** [review]
fixes: #3807

**Comment by xcodz-dot** [comment]
> fixes: #3807

not here, edit your top PR body

**Review by dhruvmanila — APPROVED** [review]

**Pull request merged**